main/guides/getting-started/how-to-use-ui-kit.md

```diff
@@ -0,0 +1,147 @@
+
+for await (const purses of subscribeLatest(pursesNotifier)) {
+  console.log('Got purses:', purses);
+}
+```
+
+### Using a Custom Signer
+If you're using a custom signer instead of Keplr, you can provide your own signer implementation when creating the `AgoricWalletConnection`. Here's an example of setting up a custom signer using the `@cosmjs/stargate` library:
+
+```js
+import {
+  agoricRegistryTypes,
+  agoricConverters,
+  makeAgoricWalletConnection,
+} from '@agoric/web-components';
+import { Registry } from '@cosmjs/proto-signing';
+import {
+  AminoTypes,
+  defaultRegistryTypes,
+  createBankAminoConverters,
+  createAuthzAminoConverters,
+} from '@cosmjs/stargate';
+
+// ...
+
+const signingStargateClient = await SigningStargateClient.connectWithSigner(
+  rpcEndpoint, // RPC endpoint to use
+  customAminoSigner, // E.g. window.getOfflineSignerOnlyAmino(chainId)
+  {
+    aminoTypes: new AminoTypes({
+      ...agoricConverters,
+      ...createBankAminoConverters(),
+      ...createAuthzAminoConverters(),
+    }),
+    registry: new Registry([...defaultRegistryTypes, ...agoricRegistryTypes]),
+  },
+);
+const agoricWalletConnection = await makeAgoricWalletConnection(
+  chainStorageWatcher,
+  rpcEndpoint,
+  (e: unknown) => {
+    console.error('wallet connection error', e);
+  },
+  { address: myAddress, client: signingStargateClient },
+);
+```
+
+## Executing Offers
+The UI Kit provides functionality to execute offers (transactions) on the Agoric blockchain. The `makeOffer` function allows you to create and submit an offer, specifying the contract instance, the amounts to give and want, and any additional arguments. You can also provide a callback function to handle the different stages of the offer execution process.
+
+```js
+import { makeAgoricChainStorageWatcher } from '@agoric/rpc';
+import { makeAgoricWalletConnection } from '@agoric/web-components';
+
+const watcher = makeAgoricChainStorageWatcher(apiAddr, chainName);
+const connection = await makeAgoricWalletConnection(watcher, rpcAddr);
+
+const amountToGive = { brand: someBrand, value: 123n };
+const amountToWant = { brand: someOtherBrand, value: 456n };
+
+connection.makeOffer(
+  {
+    source: 'agoricContract',
+    instancePath: ['SimpleSwapExampleInstance'],
+    callPipe: [
+      ['getSwapManagerForBrand', [amountToGive.brand]],
+      ['makeSwapOffer'],
+    ],
+  },
+  {
+    give: { In: amountToGive },
+    want: { Out: amountToWant },
+  },
+  { exampleArg: 'foo' },
+  ({ status, data }) => {
+    if (status === 'error') {
+      console.error('Offer error', data);
+    }
+    if (status === 'seated') {
+      console.log('Transaction submitted:', data.txn);
+      console.log('Offer id:', data.offerId);
+    }
+    if (status === 'refunded') {
+      console.log('Offer refunded');
+    }
+    if (status === 'accepted') {
+      console.log('Offer accepted');
+    }
+  },
+);
+```
+
+## Conclusion
```

Comment:
a Conclusion makes sense for a thesis or essay. It seems out of place to me in HOW-TO

Reply:
Conclusion removed